frank/fit.py

```diff
@@ -378,11 +386,78 @@ def perform_fit(u, v, vis, weights, geom, model):
         return [sol, None]
 
 
-def output_results(u, v, vis, weights, sol, iteration_diagnostics, model):
+def run_multiple_fits(u, v, vis, weights, geom, model):
+    r"""
+    Perform and overplot multiple fits to a dataset by varying two of the
+    model hyperparameters
+
+    Parameters
+    ----------
+    u, v : array, unit = :math:`\lambda`
+        u and v coordinates of observations
+    vis : array, unit = Jy
+        Observed visibilities (complex: real + imag * 1j)
+    weights : array, unit = Jy^-2
+        Weights assigned to observed visibilities, of the form
+        :math:`1 / \sigma^2`
+    geom : SourceGeometry object
+        Fitted geometry (see frank.geometry.SourceGeometry)
+    model : dict
+        Dictionary containing model parameters the fits use
+
+    Returns
+    -------
+    multifit_fig : Matplotlib `.Figure` instance
+        All produced figures, including the GridSpecs
+    multifit_axes : Matplotlib `~.axes.Axes` class
+        Axes for each of the produced figures
+    """
+
+    logging.info(' Looping fits over the hyperparameters `alpha` and `wsmooth`')
+    alphas = model['hyperparameters']['alpha']
+    ws = model['hyperparameters']['wsmooth']
+    sols = []
+
+    def float_to_list(x):
+        if type(x) is float:
+            return [x]
+        return x
```

Comment:
I think we should accept ints here too (e.g. alpha=1 or w_smooth=0):

```suggestion
    def float_to_list(x):
        if np.isscalar(x):
            return [x]
        return x
```

Reply:
good point, I did this and changed the function name to `number_to_list`